_data/packages.yml

```diff
@@ -120,6 +120,7 @@
   sourceSize: 7
```

Comment:
Does the package size need to be updated here?

Reply:
Leaving as-is for now since we did not change it when we attempted to reduce the package size